src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -244,11 +257,21 @@ export class LogsPageComponent implements OnInit {
       traceIdForSummary = this.formatTraceForSummary(log.trace);
     }
 
-    if (log.message) {
+    if (log.message && this.isAdmin) {
       summary = log.message;
       payload = this.formatTextPayloadForDisplay(log.message);
     } else if (log.details) {
       payload = log.details;
+
+      if (!this.isAdmin) {
```

Comment:
All these information hiding is not necessary.
- If back-end failed to hide/encrypt the necessary information to be hidden/encrypted, then they're still there in the API response. Hiding from view is not hiding it at all.
- If back-end did hide/encrypt those as expected, this entire additional processing is pointless.

Reply:
Understood. It is updated.